config/rubocop.yml

```diff
@@ -58,6 +63,9 @@ Naming/PredicateName:
     - is_
     - have_
 
+Rails/CreateTableWithTimestamps:
+  Enabled: false
```

Comment:
Rails migration generator adds timestamps by default, so if they have been removed, it was intentional. Not sure if we need this cop.

Reply:
👍  for disabling it

Reply:
I usually make a decision whether I need them or not and if it turns out we need them later, we can still add them. So, I don't think we need it, either.